### ISSUE TITLE
Wind Barbs

### ISSUE THREAD

**Issue by itsgifnotjiff** [holoviz/geoviews#720]
#### Is your feature request related to a problem? Please describe.
One of the few things missing from GeoViews are domain specific plots out of which the most commonly used one is the Wind Barbs one. Matplotlib, MetPy and others allow for static and WMS services can serve dynamic Wind Barbs but using a VectorField it should (maybe already is) be possible to put some Wind barbs on a GeoViews map object. 

![image](https://github.com/holoviz/geoviews/assets/50274281/c9797b56-d659-4f53-b3bd-eb572ee1605a)

#### Describe the solution you'd like
Ideally we would replicate the [Matplotlib API](https://matplotlib.org/stable/gallery/images_contours_and_fields/barb_demo.html) where we allow to obviously define the x,y,u,v and c but also allow for some options like density and such.

#### Describe alternatives you've considered
I have experimented with the VectorField examples i've found around Discourse and StackOverflow, but right now I am using Matplotlib that I wrap in `panel.pane.Matplotlib`.

**Comment by ahuang11** [comment]
There is a matplotlib backend barbs https://geoviews.org/gallery/matplotlib/wind_barbs_example.html

**Comment by itsgifnotjiff** [comment]
Correct!! I would like to have a dynamic VectorField Geoviews one since the Matplotlib barbs one is trivial to implement but static.

**Comment by wx4stg** [comment]
I'm interested in contributing this but it feels more apt for holoviews, not geoviews. (Wind barbs are nice for maps, but what if I run [CM1](https://github.com/george-bryan/CM1) or something and want to visualize its output?)

**Comment by itsgifnotjiff** [comment]
Wind barbs are a geospatial plot and that's why I thought geoviews is the right package for it. Mr. @hoxbro what do you think?
